src/core/stream.c

```diff
@@ -713,7 +725,79 @@ QuicStreamParamGet(
 
         Status = QUIC_STATUS_SUCCESS;
         break;
+    case QUIC_PARAM_STREAM_BLOCKED_TIMINGS:
```

Comment:
nit: add a new line before this, and just go ahead and wrap this case with `{ }` because you'd declaring variables below.